lib/mix/lib/mix/project_stack.ex

```diff
@@ -2,188 +2,201 @@ defmodule Mix.ProjectStack do
   @moduledoc false
 
   use Agent
-  @timeout 30_000
+  @timeout 30000
```

Comment:
I think it would be nice if the formatter didn't remove underscores from numbers.

Reply:
It formats with underscores for numbers with >= 6 digits.